vite.config.js

```diff
@@ -5,7 +5,14 @@ import {vanillaExtractPlugin} from '@vanilla-extract/vite-plugin';
 import noBundlePlugin from 'vite-plugin-no-bundle';
 
 export default defineConfig({
-    plugins: [react(), vanillaExtractPlugin(), noBundlePlugin()],
+    plugins: [
+        react(),
+        vanillaExtractPlugin(),
+        noBundlePlugin({
+            // Change .css.js files to something else so that they don't get re-processed by consumer apps using vanilla extract too
+            fileNames: ({name}) => `${name.replace(/\.css$/, '.css.vanilla')}.js`,
```

Comment:
Vanilla Extract Next plugin applies to node_modules too, so it tries to re-process mistica files (which are already compiled by mistica build) and result in errors. With this change, we make those files invisible for the Next plugin in apps using mistica and VE (zeus, for example).

Reply:
perhaps a better name would be `.css.mistica`